src/dsp_tools/xmllib/models/licenses/other.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from dsp_tools.xmllib.models.licenses.recommended import License
+
+
+class LicenseOther:
+    """
+    Pre-defined licenses that are available in DSP.
+    If the user is free to choose a license, we recommend to chose from our [recommended licenses.](https://docs.dasch.swiss/latest/DSP-TOOLS/xmllib-api-reference/licenses/recommended/)
+
+    - `PublicDomains`: [See `PublicDomains` for details.](https://docs.dasch.swiss/latest/DSP-TOOLS/xmllib-api-reference/licenses/other/#xmllib.models.licenses.other.PublicDomains)
```

Comment:
I'm a bit reluctant with the name "PublicDomains". It is too similar to "PublicDomain". What about "public"?